src/cse-machine/interpreter.ts

```diff
@@ -107,12 +107,17 @@ export class Control extends Stack<ControlItem> {
   }
 
   public canAvoidEnvInstr(): boolean {
-    return this.numEnvDependentItems === 0
+    return this.numEnvDependentItems == 0
```

Comment:
Why was this changed from `===` to `==`?

Reply:
That was unintentional, I'll change it back